src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -36,30 +37,50 @@ class SensorFusion : public Subject<World>,
     SensorFusion(const SensorFusion &)            = delete;
 
    private:
-    void onValueReceived(RefboxData refbox_data) override;
-    void onValueReceived(RobotStatus robot_status) override;
-    void onValueReceived(VisionDetection vision_detection) override;
+    void onValueReceived(SensorMsg sensor_msg) override;
+
+    /**
+     * Updates world based on a new SensorMsg
+     *
+     * @param sensor_msg new SensorMsg
+     */
+    void updateWorld(const SensorMsg &sensor_msg);
 
     /**
-     * Updates world based on new refbox data
+     * Updates world based on a new SensorMsg
      *
-     * @param refbox_data new refbox data
+     * @param sensor_msg new SensorMsg
```

Comment:
Nit this kind of violates the style guide:
"Functions that perform similar tasks should be commented as a group if it improves clarity."

Can we clump all `updateWorld`s together?

Reply:
donezo